Source/ocdm/playLevel.h

```diff
@@ -0,0 +1,71 @@
+ /*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "Module.h"
+
+using namespace WPEFramework::Core ;
```

Comment:
The whole idea of the exercise was to remove the Core dependency from Netflix. To be able to call the new C function you need the PlayLevels struct declaration, will include this header file and get the Core dependency again for free again :) And now also the C header file contains C++ code...
What I propose is to declare this struct just in the open_cdm_ext.h header file and remove all the core references (and so also the PlayLevelsJSON must be removed, it will just be a simple struct with data fields).
Then  the PlayLevelsJSON class can just be declared inside the method that implements the API call. That way we completely decouple the C interface from any Core functionality